Examples/KerrBH/params_cheap.txt

```diff
@@ -0,0 +1,106 @@
+# Thresholds on the change across a cell which prompts regrid
+regrid_threshold = 0.2
+
+# Level data
+# Maximum number of times you can regrid above coarsest level
+max_level = 3 #4 There are (max_level+1) grids, so min is zero
+# Frequency of regridding at each level
+# Need one for each level, ie max_level+1 items
+# Generally you do not need to regrid frequently on every level
+regrid_interval = 50 25 5 5 5 5 5
+# Max box sizes
+max_grid_size = 16
+# Min box size
+block_factor = 16
+# Determines how exactly regridding tries to fit the tagging
+fill_ratio = 0.75
+
+#boundaries and periodicity of grid
+#Periodic directions - 0 = false, 1 = true
+isPeriodic = 0 0 0
+# if not periodic, then specify the boundary type
+# 0 = static, 1 = sommerfeld, 2 = reflective
+# (see BoundaryConditions.hpp for details)
+hi_boundary = 1 1 1
+lo_boundary = 1 1 2
+
+# if reflective boundaries selected, must set
+# parity of all vars (in order given by UserVariables.hpp)
+# 0 = even
+# 1,2,3 = odd x, y, z
+# 4,5,6 = odd xy, yz, xz
+vars_parity            = 0 0 4 6 0 5 0    #chi and hij
+                         0 0 4 6 0 5 0    #K and Aij
+                         0 1 2 3          #Theta and Gamma
+                         0 1 2 3 1 2 3    #lapse shift and B
+                         0 1 2 3          #Ham and Mom
+                         0 0              #ADM M and J
+
+# if sommerfeld boundaries selected, must select 
+# asymptotic values (in order given by UserVariables.hpp)
+vars_asymptotic_values = 1.0 1.0 0.0 0.0 1.0 0.0 1.0 #chi and hij
+                         0.0 0.0 0.0 0.0 0.0 0.0 0.0 #K and Aij
+                         0.0 0.0 0.0 0.0             #Theta and Gamma
+                         1.0 0.0 0.0 0.0 0.0 0.0 0.0 #lapse shift and B
+                         0.0 0.0 0.0 0.0             #Ham and Mom
+                         0.0 0.0                     #ADM M and J (wrong for J, but not necessary)
+
+# Set up time steps
+# dt will be dx*dt_multiplier on each grid level
+# HDF5files are written every dt = L/N*dt_multiplier*checkpoint_interval
+checkpoint_interval = 1
+plot_interval = 0
+num_plot_vars = 4
+plot_vars = chi K lapse shift1
+dt_multiplier = 0.25
+stop_time = 10.0
+max_steps = 2
+
+#Lapse evolution
+lapse_power = 1.0
+lapse_coeff = 2.0
+lapse_advec_coeff = 1 # 1 makes the lapse gauge 1+log slicing
+
+# Shift evolution coefficients
+shift_advec_coeff = 0 # Usually no advection for beta
+shift_Gamma_coeff = 0.75 # 
+eta = 1.0 # This is beta_driver, usually of order 1/M_ADM of spacetime
+
+# CCZ4 parameters
+# if using BSSN the kappa values should be zero
+formulation = 1  # 1 for BSSN, 0 for CCZ4
+kappa1 = 0
+kappa2 = 0
+kappa3 = 0
+covariantZ4 = 0 # 0: default. 1: dampk1 -> dampk1/lapse
+
+# coefficient for KO numerical dissipation
+# NB must be less than 0.5 for stability
+sigma = 0.3
+
+#extraction params
+activate_extraction = 0
+num_extraction_radii = 3
+extraction_radii = 30. 40. 50.
+extraction_levels = 1 1 0
+num_points_phi = 50
```

Comment:
I think it would be nice to also update the params.txt file so it has extraction and they do roughly the same thing but params_cheap.txt has fewer levels and no extraction (mirroring the BinaryBH example). The current params.txt has periodic BCs which looks a bit rubbish :-) This is not really related to this PR but it seems a good time to fix it.

(Also minor point, num_plot_vars here can be set to 0 as plot files are turned off, and the extraction params could also be set to minimal levels.)

Reply:
I will improve this!
I'd like to leave the extraction in the cheap params though. It is how I did many tests and found it extremely useful. Why? Because even in cheap resolution, extraction of the mass is really accurate (and quick, so no problem). Do you agree?

Reply:
Sure, I'm ok with this if it helps testing.